monitoring.md

```diff
@@ -0,0 +1,213 @@
+
+The first endpoint provides application-level statistics, such as database
+performance, replication latencies, topology info, etc. These are the same
+metrics that are visible in the [UI Internals
+tab](/operations.md#internals-tab) Stats pane under the `snowth.` namespace.
+
+The second endpoint provides libmtev framework statistics, such as job queue
+latencies, memory management, and REST API latencies. These are the same
+metrics that are visible in the [UI Internals
+tab](/operations.md#internals-tab) Stats pane under the `mtev.` namespace.
+
+The format options are discussed below.
+
+> Changing an existing check against the default format to tagged format, or
+> vice versa, will result in different metric names, even though the data
+> represented is the same.
+
+### JSON Default Format
+
+The default format for metric names is hierarchical. The broadest category of
+statistics is the top level, descending to more specific sub-categories, and
+finally listing individual metrics.
+
+For example, the raw database PUT latency histogram metric
+is represented in the default format as:
+
+```json
+{
+  "db": {
+    "raw": {
+      "put`latency": {
+        "_type": "h",
+        "_value": [ (histogram values) ]
+      }
+    }
+  }
+}
+```
+
+which results in a metric named:
+
+```
+db`raw`put`latency
+```
+
+There are no tags in the default format.
+
+### JSON Tagged Format
+
+If provided the query string `format=tagged`, both endpoints will produce
+metrics with [stream tags](/tags.md) instead of the hierarchy used in the
+default format. The same metric from above is represented in tagged format as:
+
+```json
+{
+  "latency|ST[app:snowth,db-impl:nom,db-type:raw,operation:put,snowth-node-id:(node-uuid),units:seconds]": {
+    "_type": "h",
+    "_value": [ (histogram values) ]
+  }
+}
+```
+
+which results in a metric named `latency` with tags indicating the database
+type (raw) and the type of operation (put) that are encoded in the metric name
+in the default format. There are additional tags for the node's UUID and a
+"units" tag indicating what the metric's value represents. In this case it is
+seconds.
+
+## Monitor Module
+
+The internal monitor module exports all of the same statistics (both
+application and libmtev framework) as the JSON endpoints above. It records them
+in the tagged format (described above) under a designated account ID and check
+UUID. The module may be configured to store these metrics at intervals ranging
+from 1 second to several minutes or more.
+
+Metrics stored by the monitor module are replicated to additional nodes (if
+any) in the same way as metrics ingested from outside.
+
+The monitor module is not enabled by default. To enable it, add the following
+configuration to `/opt/circonus/etc/irondb-modules-site.conf` then restart the
+IRONdb service:
+
+```xml
+<generic image="monitor" name="monitor">
+  <config>
+    <uuid>00000000-0000-0000-0000-000000000000</uuid>
```

Comment:
I would either change this to `<check-uuid>` in there, or stick a random UUID in here for the customers that literally copy/paste this in.

Reply:
We have the all-zeroes UUID in other places within the docs. IMO it's better to have an obviously-invalid value so that people know they have to bring their own value.  I can change it if you feel strongly that it should "just work" even if one copy/pastes.